README.md

```diff
@@ -37,46 +37,46 @@ Please note: Newer Keptn versions might be compatible, but compatibility has not
 
 ## Installation
 
-The *job-executor-service* can be installed as a part of [Keptn's uniform](https://keptn.sh) using `helm`:
-
-```bash
-helm upgrade --install -n keptn job-executor-service https://github.com/keptn-contrib/job-executor-service/releases/download/<VERSION>/job-executor-service-<VERSION>.tgz
-```
-
-Please replace `<VERSION>` with the actual version you want to install from the compatibility matrix above or the 
-[GitHub releases page](https://github.com/keptn-contrib/job-executor-service/releases).
-
-
-### Installation on remote execution-plane
-
-You can install job-executor-service either in the same cluster and namespace as Keptn (see above), or on a completely 
+The *job-executor-service* can be installed as a part of [Keptn's uniform](https://keptn.sh) using `helm`.
+It is recommended to install the *job-executor-service* on the remote execution-plane, which can be on the same cluster, or on a completely
 separate Kubernetes environment (see [Keptn docs: Multi-cluster setup](https://keptn.sh/docs/0.11.x/operate/multi_cluster/) for details).
+Alternatively it is also possible to install the *job-executor-service* in the [same namespace as Keptn](#installation-in-the-same-namespace-as-keptn). 
 
-In order to do the latter, job-executor-service helm chart provides some values that can be configured:
-
-* `remoteControlPlane.enabled` - needs to be set to `true` in order to connect to the remote Keptn instance
+In order to install the *job-executor-service* on the remote execution plane, some values of the helm chart can be configured:
```

Comment:
Replace "some values of the helm chart can be configured" with "some values of the helm chart need to be configured"